src/main/java/kr/codesqaud/cafe/controller/UserLogicController.java

```diff
@@ -0,0 +1,48 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.basic.User;
+import kr.codesqaud.cafe.repository.UserMemoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/user")
+public class UserLogicController {
+    UserMemoryRepository userMemoryRepository;
+
+    @Autowired
+    public UserLogicController(UserMemoryRepository userMemoryRepository) {
+        this.userMemoryRepository = userMemoryRepository;
+    }
+
+    @PostMapping("/create")
+    public String create(@RequestParam String userId,
+                         @RequestParam String password,
+                         @RequestParam String name,
+                         @RequestParam String email,
+                         Model model
+    ) {
+        userMemoryRepository.join(new User(userId, password, name, email));
+
+        return "redirect:/user/list";
+    }
+
+    @GetMapping("/list")
+    public String list(Model model) {
+        List<User> users = userMemoryRepository.findAll();
+        model.addAttribute("users", users);
+        return "user/list";
+    }
+
+    @GetMapping("/profile/{userId}")
+    public String profile(@PathVariable String userId,
+                          Model model) {
+        User user = userMemoryRepository.findUser(userId); // null 포인트 예외 처리 필요
```

Comment:
```suggestion
        //TODO:  null 포인트 예외 처리 필요
        User user = userMemoryRepository.findUser(userId); 
```

TODO 주석을 활용해 보세요. 그리고 Optional 도 활용해 보세요.